src/api/base/index.ts

```diff
@@ -48,19 +48,46 @@ export default class BaseApi {
     }
   }
 
-  protected post(url: string, data: unknown, options: Options = {}): Promise<Response> {
-    return this.handleMethod(url, 'POST', this.fetch.bind(this), this.API_URL_GETTER, options, data)
+  protected post(
+    url: string,
+    data: unknown,
+    options: Options = {},
+    fetchFn?: 'singleEnv' | 'multipleEnvs'
+  ): Promise<Response> {
+    const params = [url, 'POST', this.fetch.bind(this), this.API_URL_GETTER, options, data]
+    if (fetchFn === 'singleEnv') {
```

Comment:
Made this distinction because we don't want to allow some POST queries to be repeated on both envs, resulting for example in orders in both prod and barn